README.md

```diff
@@ -474,26 +477,27 @@ based on evidence that it will make money in the future.
     company.
 - 🄳 The [**valuation**](https://en.wikipedia.org/wiki/Valuation_(finance)) of the company is the
   present value investors believe the company has.
-  If the company is doing well, growing revenue or has indications of future revenue (like
-  growing users or traction in a promising market), the company’s valuation will usually be
-  on the rise. That is, the **price** for an investor to buy one share of the company would be
-  increasing.
+  If the company is doing well, bringing in revenue or establishing indications of future
+  revenue (like a growing number of users or traction in a promising market), the company’s
+  valuation will usually be on the rise.
+  With that rising valuation, the **price** for an investor to buy one share of the company
```

Comment:
The **price** for an investor to buy one share of the company would be increasing.

Reply:
Why "would be increasing" in place of "increases"? I'm not seeing what the modal auxiliary adds.